ghcide/src/Development/IDE/Spans/AtPoint.hs

```diff
@@ -203,6 +205,51 @@ gotoDefinition
 gotoDefinition withHieDb getHieFile ideOpts imports srcSpans pos
   = lift $ locationsAtPoint withHieDb getHieFile ideOpts imports pos srcSpans
 
+fixityAtPoint :: HieAstResult -> HiFileResult -> HscEnv -> Position -> IO (Maybe (Maybe Range, [T.Text]))
+fixityAtPoint (HAR _ hf _ _ _) hi env pos = fmap listToMaybe $ sequence $ pointCommand hf pos fixityInfo
+  where
+    fixityInfo :: HieAST a -> IO (Maybe Range, [T.Text])
+    fixityInfo ast = do
+      let range = realSrcSpanToRange $ nodeSpan ast
+          names = mapMaybe eitherToMaybe $ M.keys $ getNodeIds ast
+          fixities = getFixityFromModIface names
+
+      -- Get fixity from HscEnv for local defined operator will crash,
+      -- we first try to use ModIface to get fixities, and then use
+      -- HscEnv if ModIface doesn't available.
```

Comment:
And the reason we can't just rely on the `ModIface` fixities is presumably because it will *only* have fixities for local definitions?

Reply:
Do the tests cover both cases?

Reply:
> And the reason we can't just rely on the ModIface fixities is presumably because it will only have fixities for local definitions?

If I remember correctly, ModIface only has definitions from packages, and we rely on ghcenv to get fixities for local definitions, I'll confirm it again and comment here.

> Do the tests cover both cases?

The test data file from line 24 is for local definitions.